lib/haskell/natural4/src/LS/XPile/MathLang/GenericMathLang/GenericMathLangAST.hs

```diff
@@ -131,21 +148,46 @@ type MdGrp = [ExpMetadata]
 ------------------------------------------------------------
 -- TODO: Look into whether the costs of using records for sum variants (eg partial functions) outweigh benefits
 
+-- | Variable names in GML.
+--
+-- The internal representation is text.
+--
 newtype Var = MkVar T.Text
   deriving stock (Show)
   deriving newtype (Eq, Hashable)
 makePrisms ''Var
 -- Add metadata like what the original L4 string was?
 -- Or do that only at the final pretty printing stage, when we normalize the formatting etc?
 
+-- | Construct a variable from its name / text.
+--
 mkVar :: T.Text -> Var
-mkVar = view $ re _MkVar
+mkVar = coerce
```

Comment:
Thanks for this; I'll try to remember to just use coerce instead of isos for this sort of thing next time!